src/Shape.js

```diff
@@ -1,6 +1,95 @@
 var ObstacleFactory = function(wallOptions){
-    // shape.prototype.draw()
+	/*cosnt */
+	const separateLineWidthRatio = 0.03;
+	const segmentMaxWdithRatio = 0.4;
+	const segmentMinWdithRatio = 0.05;
+
```

Comment:
these are hard coded values. can you make them all to " this.startPointY = wallOptions.y;" form